src/DTO/addressDTO.ts

```diff
@@ -0,0 +1,19 @@
+export interface AddressResponseDTO{
+    addressId: number;
+    city: string; 
+    zipcode: string;
+    address: string;
+}
+
+export interface AddressRequestDTO{
+    addressId: number;
```

Comment:
Consider making addressId optional, that way you don't need AddressRequestDTO2
```suggestion
    addressId?: number;
```